snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -165,3 +165,330 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.3375.2.1.1.2.9.29
       name: sysClientsslStatHandshakeFailures
+
+  # Metrics from F5-BIGIP-LOCAL-MIB provide monitoring information about virtual servers, nodes, and pools.
+  # These concepts are related to the LTM (Local Traffic Management) features of BIG IP devices.
+  # TL;DR:
+  # * Virtual servers act as load balancers that are assigned pools of nodes (= servers).
+  # * Clients can use a virtual server as an entrypoint to an application service that spans multiple machines.
+  # Learn more: https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-basics-11-6-0.html
+
+  # Virtual servers.
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.1
+      name: ltmVirtualServNumber
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.2
+      name: ltmVirtualServTable
+    symbols:
+      # Gauges
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.9
+        name: ltmVirtualServEnabled
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.10
+        name: ltmVirtualServConnLimit
+    metric_tags:
+      - tag: server
+        column:
+          OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.1
+          name: ltmVirtualServName
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.2.3
+      name: ltmVirtualServStatTable
+    symbols:
+      # Counters (= rates)
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.6
+        name: ltmVirtualServStatClientPktsIn
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.7
+        name: ltmVirtualServStatClientBytesIn
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.8
+        name: ltmVirtualServStatClientPktsOut
+      - OID: 1.3.6.1.4.1.3375.2.2.10.2.3.1.9
+        name: ltmVirtualServStatClientBytesOut
```

Comment:
@therve Interested in your opinion on this: these are `Counter64`, so they'll be submitted as rates, and I don't think we need them as monotonic counts (because we discover that was broken in the UI) so I haven't used `forced_type: monotonic_count_and_rate`. Does that sound fair?

Reply:
Offline discussion - submitting `rate` should be preferred. So current state of this PR is OK.